app/models/spree/stock_location.rb

```diff
@@ -3,6 +3,7 @@
 module Spree
   class StockLocation < ApplicationRecord
     self.belongs_to_required_by_default = false
+    self.ignored_columns += [:backorderable_default]
```

Comment:
Should we just drop the column if it's not used ?

Reply:
It's better to do that in a second step in another release. When you delete a column from the database during deployment then the currently running code still refers to it and fails until the new code is loaded. That's why you can ignore a column first. After that is deployed, it's safe to remove the column.